torchao/experimental/tests/test_int8_dynamic_activation_intx_weight.py

```diff
@@ -370,7 +355,7 @@ def test_export_QDQLayout(self):
             "torch.ops.torchao.choose_qparams_affine.default(input_1, 'ASYMMETRIC', [1, 512], torch.int8, None, None, None, torch.float64, torch.int64)",
```

Comment:
nit: I think the test should probably just be checking the ops unless strictly necessary, otherwise the test might be a bit fragile, there is no guarantee on the names of intermediate values in fx graph